src/ebay-listbox-button/listbox-button.tsx

```diff
@@ -130,7 +130,9 @@ const ListboxButton: FC<EbayListboxButtonProps> = ({
     const onButtonClick = () => {
         setExpanded(!expanded)
         setOptionsOpened(true)
-        setTimeout(() => optionsContainerRef.current.focus(), 0)
+        setTimeout(() => optionsContainerRef.current.focus({
```

Comment:
Can you move it to a `focusOptionsContainer()` function and use it also on the [`Enter` case](https://github.com/eBay/ebayui-core-react/blob/master/src/ebay-listbox-button/listbox-button.tsx#L142)